athf/core/research_manager.py

```
@@ -523,6 +523,115 @@ def link_hunt_to_research(self, research_id: str, hunt_id: str) -> bool:
+            with open(file_path, "r", encoding="utf-8") as f:
+                content = f.read()
+        except OSError:
+            return None
+
+        # Update frontmatter with structured hypothesis metadata
+        new_frontmatter = dict(research_data.get("frontmatter", {}))
+        new_frontmatter["generated_hypothesis"] = {
+            "hypothesis": hypothesis,
+            "mitre_techniques": list(mitre_techniques or []),
+            "data_sources": list(data_sources or []),
+            "generated_at": datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ"),
```

Ruff (0.15.13)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)